src/core/frameworks/gql/queries.ts

```diff
@@ -223,18 +226,35 @@ export const GET_LINE_PAGE_QUERY = gql`
   }
 `;
 
-export const GET_LINE_PAGE_AUX_QUERY = gql`
+export const GET_LINE_EVENTS_QUERY = gql`
   ${LINE_EVENT_FRAGMENT}
+  ${TOKEN_FRAGMENT}
+  ${ESCROW_EVENT_FRAGMENT}
   ${SPIGOT_EVENT_FRAGMENT}
 
-  query getLinePageAux($id: ID) {
+  query getLineEvents($id: ID) {
     lineOfCredit(id: $id) {
-      events(first: 20) {
+      events {
         ...LineEventFrag
       }
 
+      escrow {
```

Comment:
Only want events not other data like deposits

Reply:
`events` are nested within `escrow.deposits` which is why we need to pull more data. 

```
const ESCROW_FRAGMENT = gql`
  ${TOKEN_FRAGMENT}
  ${ESCROW_EVENT_FRAGMENT}
  fragment EscrowFrag on Escrow {
    id
    minCRatio
    deposits {
      amount
      enabled
      token {
        ...TokenFrag
      }
      events {
        ...EscrowEventFrag
      }
    }
  }
`;
```

Reply:
u can pull just the events without all the other data tho
` escrow { deposits { events { ...EscrowEventFrag } } }`

Reply:
as discussed, this is being used to populate data for line page, so this is going to remain the same